website/content/en/docs/setup/_index.md

```diff
@@ -25,3 +25,8 @@ In order to cover as many observability use cases as possible, we've built Vecto
 Vector is built to be deployable in multiple roles and topologies:
 
 {{< jump "/docs/setup/deployment" >}}
+
+## Download Observability Pipelines
+
+[Observability Pipelines](https://www.datadoghq.com/product/observability-pipelines/) is a commercial aggregator pipeline.
+
```

Comment:
This page previously included 3 logical steps for setting up Vector that were ordered sequentially:
1. Installing
2. Configuring
3. Deploying

We are now adding a fourth header, which is not the next logical step after "Deploying" in the setup process, which can be confusing to readers. 

Rather, the set up process for Vector and OP are distinct processes. My two cents would be to add this content and link to OP at the top of this page, as the reader needs to first choose if they would like to continue with Vector or with OP. What do you think?

Reply:
That makes sense, how about mentioning OP in the first paragraph? So something like:

> This section of the documentation covers everything you need to get up and running with Vector, from installation to deployment. Alternatively, download Observability Pipelines if you want to build, manage, and monitor your pipelines in the Datadog UI.

And link "Observability Pipelines" to https://www.datadoghq.com/product/observability-pipelines/.

@barieom

Reply:
I like your proposal May, let's go with that!

Reply:
Updated!